api/v1beta2/foundationdbcluster_types.go

```diff
@@ -1735,6 +1739,8 @@ type BuggifyConfig struct {
 
 	// CrashLoopContainers defines a list of process group IDs and containers
 	// that should be put into a crash looping state.
+	// +kubebuilder:validation:MinLength=0
+	// +kubebuilder:validation:MaxLength=8
```

Comment:
Those are the wrong annotations:

```text
+kubebuilder:validation:MaxLength
int
specifies the maximum length for this string.
```

Since this is a slice you have to use `+kubebuilder:validation:MaxItems` and `+kubebuilder:validation:MinItems`